### DISCUSSION TITLE
Working Caddyfile?

### DISCUSSION THREAD

**Discussion by afsabresfan** [Donkie/Spoolman#447] [Q&A] [answered]
Can anyone share a working caddyfile configuration? I have Caddy set up on a separate host from Spoolman and I’m just not getting how to write the reverse proxy configuration. I’m sure it’s a me issue since everyone seems to say Caddy is super easy to use. I don’t have a domain; I’m using all internal IP addresses. Thanks!

**Comment by danb35** [comment, marked as answer]
If you're just using an internal IP address, I'm not sure what value the reverse proxy would bring--with or without it, you're browsing to an IP address, you aren't getting HTTPS, etc.  But with that said, this should work:
```
:80 {
    reverse_proxy ip_of_spoolman:7912
}
```
That will take any traffic from port 80 on your Caddy host and proxy it to the Spoolman instance.

**Reply by afsabresfan** [reply]
Thanks! I wanted to at least get the reverse proxy running first before trying to add the HTTPS so I can use the QR code. Not sure why I am struggling with the setup so much…seems so simple!
